jovian/jovian_nb_ext/main.js

```diff
@@ -70,19 +73,25 @@ define([
             ")\n";
         }
 
-        const jvn_commit =
-          "from jovian import commit\n" +
-          "import io\n" +
-          "from contextlib import redirect_stdout\n" +
-          "f = io.StringIO()\n" +
-          "with redirect_stdout(f):\n" +
-          "\t" +
-          commit +
-          "out = f.getvalue().splitlines()[-1]\n" +
-          "if(out.split()[1] == 'Committed'):\n" +
-          "\tprint(out.split()[-1])\n" +
-          "else:\n" +
-          "\tprint(out)";
+        const jvn_commit = `
+        from contextlib import redirect_stdout, redirect_stderr
```

Comment:
better remove this indentation inside the template string, as it may cause issues with the Python interpreter

Reply:
in the first line? Its taken in as a new line. Anyways ill move it to the previous line

Reply:
the first line is not a problem, as it simply adds a newline character

Reply:
Yes, then which indent are you referring to ?

Reply:
Ohh do you mean to indent in js and want me to revert to something like this?
![image](https://user-images.githubusercontent.com/24864161/81992193-8de76380-9660-11ea-86e4-833f2bbe9583.png)

This is why i was using string concats with \t and \n initially . So that visual flow in js is not disrupted.

I dont think it will cause a issue as Jupyter takes care of those indents
![image](https://user-images.githubusercontent.com/24864161/81992765-ed923e80-9661-11ea-8b2b-18e1b6a5642b.png)

Let me know which would be better 
1) Retain as is (since jupyter handles that)
2) Remove indents in js to be on safer side

Reply:
I would recommend 2, just to be on the safer side.